kitchen.dokken.yml

```
@@ -1,8 +1,8 @@
+---
 driver:
   name: dokken
-  privileged: true  # because Docker and SystemD
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
-  chef_license: accept-no-persist
+  privileged: true
+  chef_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
```

Let's use `current` so we can catch upstream issues more quickly.
```suggestion
  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
```

---

Isn't current behind latest? Or have I got that backwards?

---

Apparently so